src/Turbo/doc/index.rst

```diff
@@ -432,14 +430,16 @@ Now, create a "fresh" form and pass it into your stream:
 
 Now, in your stream template, "replace" the entire form:
 
-.. code-block:: html+twig
+.. code-block:: diff
 
-    {# success.stream.html.twig #}
-    <turbo-stream action="replace" target="task-form">
-        <template>
-            {{ include('task/_form.html.twig') }}
-        </template>
-    </turbo-stream>
+    {# new.html.twig #}
+     {% block success_stream %}
+    +<turbo-stream action="replace" targets="form[name=task]">
+    +    <template>
+    +        {{ block('task_form') }}
+    +    </template>
+    +</turbo-stream>
+     <turbo-stream action="replace" targets="#my_div_id">
```

Comment:
Missing the closing block and this last turbo-stream looks extra to me

Reply:
well actually, this is a diff that applies to the snippet on L366, so yes, the endblock is missing and this line is here because this is just a subset of the full file